recipes/cube_helper/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "cube_helper" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MetOffice/cube_helper/archive/v{{ version }}.tar.gz
```

Comment:
We could use the pypi tarball, but the license file isn't currently included there. I've opened a PR to fix this in future releases so it's an option https://github.com/MetOffice/cube_helper/pull/45

Reply:
@synapticarbors thanks for spotting that. I've merged MetOffice/cube_helper#45 and pushed v2.1.1 to PyPI. I may as well fix this now, but I can't figure the URL to get v2.1.1 from PyPI and so it's still getting it from GitHub. If you can suggest the form of the PyPI URL then I'm happy to push a new version of meta.yml with that in. I'll keep on Googling in the meantime.

Reply:
I believe you can just use the canonical form that's listed in the example `meta.yaml`:

`url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz`

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L14

Reply:
@synapticarbors thanks for that. 17cf32e adds that so that everything's now coming from PyPI and all builds have succeeded. Thanks for your assistance!